### ISSUE TITLE
Please just dump all the stupid Copilot AI stuff and support WSA back!!!!

### ISSUE THREAD

**Issue by AgrMayank** [microsoft/WSA#546]
### Describe the solution you'd like

WSA is massively more useful than CoPilot (which can be accessed from web as well)

It feels like Microsoft is slowly destroying all the good stuff from Windows in pursuing AI, AI and AAAIIIIIII!!! 

Yes, I know AI matters but still, don't just give up on one of the best features of windows like that!! AI is NOT and WON'T be everything.

### Please specify the version of Windows Subsystem for Android

2210.40000.7.0

**Comment by Slion** [comment]
I'm with you.

**Comment by RFLEpicGamer** [comment]
I miss pre-2023 microsoft

**Comment by MikoHaku** [comment]
I'm with you.

**Comment by the-moog** [comment]
This news is a shame but hardly unexpected.
WSA was the only Win11 carrot and even then still not enough to make me switch from Win10, given how intrusive Win11 is.

I only run a handful of Windows-only apps and had already started the process of de-Microsofting to add to my de-Googling efforts.  So byebye M$, Hello Proxmox....
I expect this news will do wonders for the uptake of Linux or Mac as a serious competitor to Microsoft, encouraging commercial developers to better support those platforms - Fingers crossed!

In my opinion, I'd add that I find in incredulous to declare poverty being the only reason to drop WSA after literally spending billions on the latest dot com bubble, I expect that same amount would cover WSA development into the next millenium. Slightly worryingly, I wonder how soon it will be before VSCode goes the same way?  M$ can't make much from that either?

This anouncement is like a bilionare announcing everybody to come round for free pizza then scuttling off in their limmo before the bill is paid - Big up the feature announcement then quietly step out the back door and keep the money instead of backing that claim up with your wallet.

I expect this to be another '20/20 hindsight vision' business decisions that will be examined in detail by academics in years to come and probably the beginning of the end for the current CEO, once the bubble bursts. As it always does...

Microsoft, like Google, Tesla, etc, are all renowned for sudden and tangential announcements and this is just another big corporation failing to see the horizon.  I guess shareholders care little about customers and just want the quick buck, forgetting that an OS is almost a lifelong choice and something people don't switch to/from that easily.  Look at the W11 growth vs previous population, it's rather poor and this is just one more reason to keep it like that.

**Comment by AgrMayank** [comment]
> This news is a shame but hardly unexpected. WSA was the only Win11 carrot and even then still not enough to make me switch from Win10, given how intrusive Win11 is.
> 
> I only run a handful of Windows-only apps and had already started the process of de-Microsofting to add to my de-Googling efforts. So byebye M$, Hello Proxmox.... I expect this news will do wonders for the uptake of Linux or Mac as a serious competitor to Microsoft, encouraging commercial developers to better support those platforms - Fingers crossed!
> 
> In my opinion, I'd add that I find in incredulous to declare poverty being the only reason to drop WSA after literally spending billions on the latest dot com bubble, I expect that same amount would cover WSA development into the next millenium. Slightly worryingly, I wonder how soon it will be before VSCode goes the same way? M$ can't make much from that either?
> 
> This anouncement is like a bilionare announcing everybody to come round for free pizza then scuttling off in their limmo before the bill is paid - Big up the feature announcement then quietly step out the back door and keep the money instead of backing that claim up with your wallet.
> 
> I expect this to be another '20/20 hindsight vision' business decisions that will be examined in detail by academics in years to come and probably the beginning of the end for the current CEO, once the bubble bursts. As it always does...
> 
> Microsoft, like Google, Tesla, etc, are all renowned for sudden and tangential announcements and this is just another big corporation failing to see the horizon. I guess shareholders care little about customers and just want the quick buck, forgetting that an OS is almost a lifelong choice and something people don't switch to/from that easily. Look at the W11 growth vs the previous population, it's rather poor and this is just one more reason to keep it like that.

That's true, it could be the shareholders messing with the decisions for profit or just plain cost-cutting. It's a stupid move considering how good WSA actually has become over the years and one of the highlight features of Win 11. 

Another POV could be an issue from Amazon's side for the depreciation of their app store. Maybe MS doesn't want people to sideload APKs (which is what most people do btw). 

Anyways this sucks that MS is slowly becoming another Google in killing features left and right.

**Comment by Foxy-null** [comment]
I'm with you too!

**Comment by lyc280705** [comment]
I can not understand why MS killed it before the release of Copilot+PC running on arm.

**Comment by ghost** [comment]
Please Microsoft! We need WSA

**Comment by z505985755** [comment]
I'm with you.

**Comment by GGuyX** [comment]
The only thing I'm missing is WSA - However, I don't at all mind generative AI stuff - As in, the additions to apps like Snip & Sketch are great, but changes to apps such as Paint, at least if it's at the cost of the ribbon not having copying/pasting functions anymore, not so much. Also, Copilot app itself can go screw itself.

**Comment by AgrMayank** [comment]
> I don't at all mind generative AI stuff - As in, the additions to apps like Snip & Sketch are great, but changes to apps such as Paint, at least if it's at the cost of the ribbon not having copying/pasting functions anymore, not so much. Also, Copilot app itself can go screw itself.

True, while I don't mind having them, especially if it's on device AI and not another subscription cash grab. But again, not at the expense of already well built tools like WSA that make life soooooo much easier.

**Comment by slmgr-vbs** [comment]
I agree all of you! Microsoft is always trying to delete useful things. Since our company started using WSA, it is certain that it's impossible to abandon WSA and use any SLOW AMD64-ONLY emulators. As a copilot+ PC user and arm-based PC user, WSA has  unique advantages. I hope Microsoft to continue developing WSA without Amazon, but with Microsoft Store filled with apks and mobile apps.
